packages/instrumenter/src/transformers/directive-bookkeeper.ts

```diff
@@ -89,4 +100,21 @@ export class DirectiveBookkeeper {
     mutatorName = mutatorName.toLowerCase();
     return this.currentIgnoreRule.findIgnoreReason(mutatorName, line);
   }
+
+  private warnAboutUnusedDirective(mutators: string[], directiveType: string, scope: string, leadingComments: types.Comment[]) {
+    for (const directive of mutators) {
+      if (directive === WILDCARD) continue;
+      if (!this.allMutatorNames.includes(directive.toLowerCase())) {
+        const directiveComment = leadingComments.find((comment) => comment.value.includes(directive));
```

Comment:
I think it is better to flow the comment through from the calling side instead of trying to search for it again here.